res/css/views/rooms/_MessageComposer.scss

```diff
@@ -104,6 +104,13 @@ limitations under the License.
     &[aria-disabled=true] {
         cursor: not-allowed;
     }
+
+    .mx_MessageComposer_noperm_error {
+        font-style: italic;
+        color: $tertiary-content;
+        font-size: $font-12px;
+        flex: 1;
+    }
```

Comment:
Why moving it within the `mx_MessageComposer_row` ? This increases the specificity of the CSS selector and was done on purpose

Reply:
Because it is only used inside there and to prevent yet another regression from happening on `mx_MessageComposer_noperm_error`.